client/lib/query-manager/test/index.js

```diff
@@ -16,6 +16,11 @@ describe( 'QueryManager', () => {
 
 	beforeEach( () => {
 		manager = new QueryManager();
+		sandbox.create();
```

Comment:
I confess I know next to nothing about sinon and this came from an SO answer that looked right and fixed my problem (clearing stubbed methods).

I tried removing the `sandbox.create()` calls in b77b103cdaa47283606bb43b94cd0b7438d7621d and tests still work fine, so it is probably just `sandbox.restore()` that's needed 👍